### PR TITLE
Stylechanges for template and application steps 1-4

### PR DESCRIPTION
**Pull request by the-daniel-rothig**
See commit messages for details

### CODE REVIEW

File: src/main/scala/controllers/ApplicationData.scala.bak [hunk 1]
```diff
@@ -0,0 +1,75 @@
+package controllers
```
Comment (jon-shipley) [inline src/main/scala/controllers/ApplicationData.scala.bak]: I don't think we need to check in a backup file.  @DougC - what do you think?

File: src/main/scala/forms/DateField.scala.bak [hunk 2]
```diff
@@ -0,0 +1,42 @@
+package forms
```
Comment (jon-shipley) [inline src/main/scala/forms/DateField.scala.bak]: Is this a backup file?  If so - we don't need to check it in.

File: src/main/twirl/views/renderers/dateField.scala.html.bak [hunk 3]
```diff
@@ -0,0 +1,26 @@
+@(field: forms.DateField)
```
Comment (jon-shipley) [inline src/main/twirl/views/renderers/dateField.scala.html.bak]: Another .bak file...

File: src/main/twirl/views/sectionForm.scala.html [hunk 4]
```diff
@@ -19,12 +22,15 @@
...
     <div class="grid-row">
         <div class="column-two-thirds">
-            <p>
-                <a href="@controllers.routes.ApplicationController.show(app.id)">&lt; Return to application overview</a>
-            </p>
-            <h2>@formatId(opportunity.id.id): @opportunity.title</h2>
-
-            <h1>@formSection.title</h1>
+            <div class="breadcrumbs">
+                <ol style="margin: 0;padding:0;">
```
Comment (jon-shipley) [inline src/main/twirl/views/sectionForm.scala.html]: I think we should consider moving this style code to a class instead.  What do you think?

**Comment by DougC** [comment]
No idea where the .bak files came from. What text editor are you using, @the-daniel-rothig? Probably best to add `*.bak` to the `.gitignore`.

**Comment by the-daniel-rothig** [comment]
My mergetool created the .baks, and they slipped under my radar in yesterday's merge headache. Will update .gitignore

**Comment by DougC** [comment]
There are still three tests failing. These are trying to extract the Question from the HTML by looking for an `<h4>`, but it appears the question is now wrapped in a `p`. Would be better if the test looked either for class of `question` or that we put a recognisable id on the element. I'll have a look at it.

**Pull request merged**